charts/fission-all/values.yaml

```diff
@@ -303,6 +303,6 @@ canaryDeployment:
 
 ## Message Queue Trigger Kind, KEDA: enable and configuration
 mqt_keda:
-  enabled: true
+  enabled: false
   connector_images:
-    kafka: fission/keda-kafka
+    kafka: fission/keda-kafka:1
```

Comment:
Let's split image and tag into separate values

Reply:
Done